themes/default/gulpfile.js

```diff
@@ -17,6 +17,9 @@ var gulp = require('gulp'),
 	concat = require('gulp-concat'),
 	uglify = require('gulp-uglify'),
 	plumber = require('gulp-plumber');
+	rework = require('gulp-rework'),
```

Comment:
Your experience here is different to mine. I've only had that happen on one project.  
 
My reasons for not wanting it to be default are: 
1. There is still manual editing required (push-pull stuff)
2. This requires the dev to self document the grid, rather than being able to rely on the styleguide default, or the pure css website for examples
3. It means someone can change the grid almost by accident - which during a project can be the cause of a lot of regressions when developers aren't diligent enough. 
4. I'm unsure why pure's grids won't do most cases. It has 5 and 24 which is more than most frameworks provide. Are we creating a lot of extra work for ourselves by not recommending a grid to the designers?